code/index.html

```diff
@@ -3,28 +3,84 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>the Studio</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+    <header>
+      <video autoplay loop muted preload="auto">
+        <source src="./yogadesktop.mp4" type="video/mp4">
+        Your browser does not support video.
+      </video>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <img class="img-tablet" src="./yogatablet.jpeg" alt="Yoga picture">
+      <img class="img-phone" src="./yogaphone.jpeg" alt="Yoga picture">
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <h1>the Studio</h1>
+    </header>
+
+    <main class="content-container">
+      <div class="content-item">
+        <form action="http://httpbin.org/anything" method="POST">
+          <h2>Sign up!</h2>
+
+          <label for="name">Name</label><br>
+          <input type="text" id="name" name="name" autocomplete="off"> 
+          <label for="e-mail">E-mail</label><br>
+          <input type="text" id="e-mail" name="e-mail" autocomplete="off"><br>
+      
+          <p>Choose your city</p>
+          <input type="radio" id="Stockholm" name="city" value="Stockholm" checked="checked">
```

Comment:
The attribute `checked` is a boolean and can therefore be written without a value. The property and the value are the same in this case (checked="checked"). When you have many attributes on one HTML element, it's good to split them up on several lines as well, like this:
```
<input
  checked
  type="radio"
  id="Stockholm"
  name="city"
  value="Stockholm">
```

Reply:
That makes the code so much easier to read!